sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/placement/AndRule.java

```diff
@@ -45,13 +47,19 @@ public EvaluationOutcome filter(Offer offer, PodInstance podInstance, Collection
             children.add(child);
         }
 
-
+        String nestedMessage = PlacementUtils.joinOutcomeMessages(children);
         if (passingCount == rules.size()) {
-            return EvaluationOutcome.pass(this, "%d of %d rules are passing:", passingCount, rules.size())
+            return EvaluationOutcome.pass(
+                    this,
+                    "%d of %d rules are passing:\n\t%s", passingCount, rules.size(), nestedMessage)
```

Comment:
there's currently recursion for recording the children (passed via `addAllChildren()` below) in `OfferEvaluator.logOutcome()`, does also logging the children here conflict with that?

Reply:
That wasn't working...  I should look into that more.